### ISSUE TITLE
Remove host.protection setting

### ISSUE THREAD

**Issue by bertfrees** [daisy/pipeline#507]
Certain TTS configuration settings are currently not possible to set dynamically (via the job's tts-config option) if `host.protection` is set to "true" (default). We should look at each of these settings and decide once and for all whether it makes sense to set them dynamically or not. If the answer is yes, always allow it, if the answer is no, always disallow it. If for none of these settings the answer is "maybe", we can remove the `host.protection` setting.

**Comment by bertfrees** [comment]
@alexbernier Do you recall what what Romain's reason for introducing the host.protection setting?

**Comment by alexbernier** [comment]
If I remember correctly : it is possible to run dtbook-to-daisy3 with a
configuration file, to set various parameters (an audio CSS file, voices
to use, etc.). Among these parameters, it is possible to set the command line
parameters for the MP3 encoders, which can potentially lead to security
issues. So, host.protected is used toallow/Disallow this kind of customization in the configuration file.

Alex

**Comment by bertfrees** [comment]
Yes, that was my understanding too. But is it needed to set things like command line parameters for MP3 encoders via the job options?

**Comment by alexbernier** [comment]
Actually, it's not really useful to set command line parameters for MP3 encoders via the job options, but what can be useful is to be able to set encoding bitrate, quality, etc. It's possible to create dedicated parameters for that, and to remove the "lame.cli.options" param.

Maybe the "host.protection" setting is also used to allow/disallow the possibility to customize the number of threads used by TTS and MP3 encoder.

Alex

**Comment by bertfrees** [comment]
Thanks for the explanation!
